### PR TITLE
A bit of spring cleaning.

### PR DESCRIPTION
**Pull request by vext01**
No functional change. Just tidying.

### CODE REVIEW

File: ykcompile/src/lib.rs [hunk 1]
```diff
@@ -467,6 +485,7 @@ impl TraceCompiler {
         CompiledTrace { mc: tc.finish() }
     }
 
+    /// Returns a pointer to the static symbol `sym`, or return an error if it cannot be found.
```
[the remark this answers is not in the document]
Comment (vext01) [reply ykcompile/src/lib.rs:488]: I think both are fine. As in `[This function] returns...`

How strongly do you feel about it? :P

[the remark this answers is not in the document]
Comment (ltratt) [reply ykcompile/src/lib.rs:488]: You can avoid the problem by not duplicating "return": "Return[s] a pointer to the static symbol `sym` or an error if it cannot be found."

[Whether it's "returns" or "return" then makes no difference and is a matter of personal taste.]

[the remark this answers is not in the document]
Comment (vext01) [reply ykcompile/src/lib.rs]: Implemented your suggestion.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
Splat, and updated PR description.

**Pull request merged**